dotnet/src/Connectors/Connectors.OpenAI/OpenAIServiceCollectionExtensions.cs

```diff
@@ -263,17 +284,26 @@ public static class OpenAIServiceCollectionExtensions
         string modelId,
         string apiKey,
         string? orgId = null,
+        string? serviceId = null) => AddOpenAIChatCompletion(services, new() { ModelId = modelId, ApiKey = apiKey, Organization = orgId }, serviceId);
```

Comment:
Same as comment above:
```suggestion
        string? serviceId = null) => AddOpenAITextGeneration(services, new() { ModelId = modelId, ApiKey = apiKey, Organization = orgId }, serviceId);
```